src/options/views/tab-settings/vm-editor.vue

```diff
@@ -37,6 +42,47 @@ export default {
       const toggled = selection === 'false' && 'true' || selection === 'true' && 'false';
       if (toggled) document.execCommand('insertText', false, toggled);
     },
+    async toggleStateHint() {
+      if (this.hint) {
+        this.hint = null;
+        return;
+      }
+      const HIDE_OPTS = [
+        // we activate only one mode: js
+        'mode',
+        // duh
+        'value',
+        // these accept only a function
+        'configureMouse',
+        'lineNumberFormatter',
+        'specialCharPlaceholder',
+      ];
+      const opts = {};
+      Object.entries({
+        ...(await import('codemirror')).defaults,
+        ...(await import('#/common/ui/code')).cmOptions,
+        ...options.get('editor'),
+      })
+      .sort(([a], [b]) => (a < b ? -1 : a > b))
```

Comment:
What's the sorting for?

Reply:
This code displays the current options object which has like 30 keys and they come in a random order. Sorting makes it easier to read.